mod/notifications/views/default/notifications/subscriptions/jsfuncs.php

```diff
@@ -4,30 +4,27 @@
 
 ?> 
 <?php //@todo JS 1.8: no ?>
-<script type="text/javascript">
+<script>
+require(['jquery'], function($) {
+	$(function () {
+		<?php
+		foreach($NOTIFICATION_HANDLERS as $method => $foo) {
```

Comment:
This would be a perfect candidate for elgg.config. This whole script could then just live as an amd module.